tests/instant_articles/test_data/input/betty-no-caption.html

```diff
@@ -0,0 +1,3 @@
+<div data-type="image" contenteditable="false" class="onion-image image inline size-big crop-original" data-image-id="9513" data-size="big" data-crop="original" data-format="jpg">
+   <div></div>
```

Comment:
I could be wrong but I think the actual input for no caption is this instead of omitted altogether:

```
<span class="caption"></span>
```